app/assets/javascripts/organisation-list-filter.js

```diff
@@ -54,6 +61,10 @@
       }
     },
 
+    countInitialDepartments: function () {
+      return $('[data-filter="item"]').length;
```

Comment:
Could un-jquery this line as well please?

```suggestion
      return document.querySelectorAll('[data-filter="item"]').length;
```

Reply:
Done, thanks.